tutorials/deploy-mdbooks-serverless-containers/index.mdx

```diff
@@ -0,0 +1,114 @@
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- Docker installed on your local machine
+
+## Why deploying on Serverless
+
+Serverless products are perfect for cost efficiency with a pay-as-you-go model and scale very well.
+
+With zero infrastructure management and many tools to integrate in CI/CD environments, `mdbooks` can benefit a lot of Serverless Containers.
+
+## Local setup
+
+1. Follow the [installation instructions](https://rust-lang.github.io/mdBook/guide/installation.html) of mdbooks.
+
+2. Run commands:
+- `mdbook init my-first-book` in order to create a sample book.
+- `cd my-first-book` to open the directory created by init command.
+
+3. *(optional)* Now you can edit the content of the book to publish.
+
+4. It is recommended to test the book using the command: `mdbook test`.
+
+## Prepare Container Registry
+
+<Message type="note">
+  We recommend using Scaleway Container Registry instead of external registries to avoid errors of rate limiting and risks regarding evolutions of CGU and pricing.
+</Message>
+
+1. Create a file named `Dockerfile` in this folder containing the following code:
+
+```dockerfile
+FROM debian:bookworm-slim
+
+WORKDIR /app
+
+# Copy necessary files (add if you have custom files)
+COPY book.toml /app/
+COPY src/* /app/src/
+
+# Install mdbooks binary
+RUN apt-get -y update; apt-get -y install curl;
+RUN mkdir /home/mdbooks
+RUN curl -sSL https://github.com/rust-lang/mdBook/releases/download/v0.4.40/mdbook-v0.4.40-x86_64-unknown-linux-gnu.tar.gz | tar -xz --directory=/home/mdbooks
+ENV PATH="$PATH:/home/mdbooks"
+
+# Serve the book:
+# 0.0.0.0 is required to listen on public interface, otherwise mdbook will only listen for localhost
+# 8080 port is used to match the Serverless Container settings.
+ENTRYPOINT ["mdbook", "serve", "-n", "0.0.0.0", "-p", "8080" ]
+```
+
+2. Open a web browser on your computer and navigate to the [Container Registry](https://console.scaleway.com/containers/) product in the Scaleway Console.
+3. Click **Create namespace**. A namespace is used to hold the image.
+
+<Message type="note">
+  A pop-up will appear with informations to log into your Container Registry Namespace with `docker login`. This will allow to push the image later.
+</Message>
+
+4. Create the namespace and save the registry endpoint. The registry endpoint looks like `rg.<region>.scw.cloud/<namespace>`
+
+<Message type="note">
+  Refer to [How to create a namespace](/docs/containers/container-registry/how-to/create-namespace/) for detailed information about Container Registry namespace creation.
+</Message>
+## Build and push
+
+1. Run the command `docker build -t rg.<region>.scw.cloud/<namespace>/<image_name>:<tag_name> .`
+* **region**: to get this information, check the namespace settings of the namespace you created.
```

Comment:
These bullet points, and basically everything that's on a new line after a numbered step (but before the next numbered step) needs to be indented two levels

Reply:
Done 😸  for my curiosity, why it's required to indent after lists ? I haven't found anything in `mxd` guidelines

Reply:
This specification directly comes from markdown (explanation [here](https://stackoverflow.com/questions/18088955/markdown-continue-numbered-list) ), otherwise it breaks the numbering :)